### PR TITLE
Add blog post about SPO v0.4.0 release

### PR DESCRIPTION
**Pull request by saschagrunert**
This is a blog post about the new Security Profiles Operrator release planned for mid December.

cc @pjbgf @jhrozek @JAORMX

### CODE REVIEW

File: content/en/blog/_posts/2021-12-13-security-profiles-operator-v0.4.0/index.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+- An OpenShift deployment manifest was included in deploy/openshift.yaml (#695, @JAORMX)
+- Bumps
+  golang.org/x/text to fix advisory GO-2021-0113 (#655, @pjbgf)
+- Log enricher now requires running auditd (`/var/log/audit/audit.log`) (#487, @saschagrunert)
+- Log libseccomp version on operator startup. (#556, @saschagrunert)
+- Removed CPU limits from SPOD and added resource requests/limits to manager and webhook. (#550, @saschagrunert)
+- The directory /etc/selinux.d used to be mounted on the hosts in previous SPO versions.
+  This is no longer the case, the directory was converted to an emptyDir instead,
+  reducing the number of required host mounts. (#698, @jhrozek)
+- The securityprofilenodestatus CR now links with the security profile its status
```
Comment (jhrozek) [inline content/en/blog/_posts/2021-12-13-security-profiles-operator-v0.4.0/index.md:105]: oops, I ended up changing the patch to use sha256 but the release note was not updated..can we change the text to say sha256?

Comment (saschagrunert) [reply content/en/blog/_posts/2021-12-13-security-profiles-operator-v0.4.0/index.md:105]: You can just change it in the PR, and I can re-generate the notes later on.

File: content/en/blog/_posts/2021-12-13-security-profiles-operator-v0.4.0/index.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+We also graduated the seccomp profile API from `v1alpha1` to `v1beta1`. This
+aligns with our overall goal to stabilize the CRD APIs over time. The only thing
+which has changed is that the seccomp profile type `Architectures` now points to
+`[]Arch` instead of `[]*Arch`.
+
+### SELinux enhancements
+
+<!--
+- Selinuxd now uses containers from quay.io/security-profiles-operator (#750, @jhrozek)
```
Comment (saschagrunert) [inline content/en/blog/_posts/2021-12-13-security-profiles-operator-v0.4.0/index.md:63]: @jhrozek do you wanna put some paragraphs about the feature here in?

File: content/en/blog/_posts/2021-12-13-security-profiles-operator-v0.4.0/index.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+#### Profile Recording
+
+<!--
+- #592, @jhrozek
+-->
+
+#### `SELinuxProfile` CRD graduation
+
+<!--
+- #675, @JAORMX
```
Comment (saschagrunert) [inline content/en/blog/_posts/2021-12-13-security-profiles-operator-v0.4.0/index.md:75]: @JAORMX do you wanna take this part?

File: content/en/blog/_posts/2021-12-13-security-profiles-operator-v0.4.0/index.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+-->
+
+### AppArmor support
+
+<!--
+- Add CRD type to represent AppArmor profiles. (#643, @pjbgf)
+- `spod` can load and unload AppArmor profiles into clusters host servers.
+  `spod` now runs as `root` and `privileged` when apparmor is enabled. (#680, @pjbgf)
+-->
```
Comment (saschagrunert) [inline content/en/blog/_posts/2021-12-13-security-profiles-operator-v0.4.0/index.md:84]: @pjbgf do you think you find some time to add content to this part?

File: content/en/blog/_posts/2021-12-13-security-profiles-operator-v0.4.0/index.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
+---
+layout: blog
+title: "What's new in Security Profiles Operator v0.4.0"
+date: 2021-12-13
```
Comment (sftim) [inline content/en/blog/_posts/2021-12-13-security-profiles-operator-v0.4.0/index.md:4]: if we can, let's pick a date that doesn't conflict with any of the post-release articles (we leave planned gaps to make that feasible).
Is the release date for SPO 0.4.0 already set?

File: content/en/blog/_posts/2021-12-13-security-profiles-operator-v0.4.0/index.md [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+<!--
+- Add CRD type to represent AppArmor profiles. (#643, @pjbgf)
+- `spod` can load and unload AppArmor profiles into clusters host servers.
+  `spod` now runs as `root` and `privileged` when apparmor is enabled. (#680, @pjbgf)
+-->
+
+### Metrics
+
+The operator now exposes metrics, which are described in detail in
+[our new metrics documentation](https://github.com/kubernetes-sigs/security-profiles-operator/blob/71b3915/installation-usage.md#using-metrics).
```
Comment (sftim) [inline content/en/blog/_posts/2021-12-13-security-profiles-operator-v0.4.0/index.md:89]:
```suggestion
our new [metrics documentation](https://github.com/kubernetes-sigs/security-profiles-operator/blob/71b3915/installation-usage.md#using-metrics).
```

**Comment by sftim** [comment]
@saschagrunert if you do change the planned date for this article / the release please sync with @karenhchu so that we can try to keep clear of other articles relating to the v1.23 release of Kubernetes - also see https://github.com/kubernetes/website/pull/30689#discussion_r764440103

File: content/en/blog/_posts/2021-12-13-security-profiles-operator-v0.4.0/index.md (hunk 5 above)
Comment (saschagrunert) [reply content/en/blog/_posts/2021-12-13-security-profiles-operator-v0.4.0/index.md:4]: No, we just decided to do it at some point next week :)

File: content/en/blog/_posts/2021-12-13-security-profiles-operator-v0.4.0/index.md (hunk 3 above)
Comment (JAORMX) [reply content/en/blog/_posts/2021-12-13-security-profiles-operator-v0.4.0/index.md:75]: let's sync next week, I can join!

File: content/en/blog/_posts/2021-12-13-security-profiles-operator-v0.4.0/index.md (hunk 5 above)
Comment (karenhchu) [reply content/en/blog/_posts/2021-12-13-security-profiles-operator-v0.4.0/index.md:4]: The post-release feature blogs are scheduled up til 12/16. What do you think about publishing this on 12/17?

Comment (saschagrunert) [reply content/en/blog/_posts/2021-12-13-security-profiles-operator-v0.4.0/index.md:4]: Sounds good to me, thank you!

**Comment by saschagrunert** [comment]
Only the AppArmor part is missing so I'll mark this as ready for review.

**Review by JAORMX — APPROVED** [review]

**Review by pjbgf — APPROVED** [review]
Just added the AppArmor section. 

All the other sections LGTM. @saschagrunert thank you for organising this. :+1:

**Comment by saschagrunert** [comment]
/assign @sftim 
for final review :)

**Comment by sftim** [comment]
/lgtm
based on https://github.com/kubernetes/website/pull/30689#pullrequestreview-832440587

/approve

**Pull request merged**